content/en/contribute/code/core/dev-environment.md

```diff
@@ -154,11 +154,9 @@ npm run local-images
 
 After the `npm` command completes successfully, a `local-build` folder will be created in the root directory of your `cht-core` project.
 
-Confirm you have these three files in the `local-build` folder: `cht-core.yml`, `cht-couchdb.yml` and `cht-couchdb-clustered.yml`
+Confirm you have these three files in the `local-build` folder: `cht-core.yml`, `cht-couchdb.yml` and `cht-couchdb-clustered.yml`.
 
-Open the project with your favorite text editor, navigate inside the `local-build` folder and update the `cht-couchdb.yml` file with the necessary ports. 
-
-In the `couchdb` configuration inside the `cht-couchdb.yml` file, right after the first `volumes` property that sets CouchDB, add the following properties and save the changes: 
+Open the project with your favorite text editor, navigate inside the `local-build` folder and update the `cht-couchdb.yml` file with the necessary ports. In the `couchdb` configuration inside the `cht-couchdb.yml` file, right after the first `volumes` property that sets CouchDB, add the following properties and save the changes: 
```

Comment:
I added the sentences in a single paragraph to clarify that they belong to the same instruction.